accounts/abi/bind/backend.go

```diff
@@ -49,7 +50,10 @@ type ContractCaller interface {
 	CodeAt(ctx context.Context, contract common.Address, blockNumber *big.Int) ([]byte, error)
 	// ContractCall executes an Ethereum contract call with the specified data as the
 	// input.
-	CallContract(ctx context.Context, call ethereum.CallMsg, blockNumber *big.Int) ([]byte, error)
+	CallContract(
+		ctx context.Context, call ethereum.CallMsg,
+		blockNumber *big.Int, overrides map[common.Address]ethapi.Account,
```

Comment:
Please don't split the signatures across lines, neither here, nor in the other instances.